boost-gradle/src/main/groovy/io/openliberty/boost/gradle/utils/GradleProjectUtil.groovy

```diff
@@ -45,13 +47,33 @@ public class GradleProjectUtil {
         Map<String, String> dependencies = new HashMap<String, String>()
 		logger.debug("Processing project for dependencies.")
 
-        project.configurations.compileClasspath.resolvedConfiguration.resolvedArtifacts.collect { it.moduleVersion.id }.each { ModuleVersionIdentifier id ->
+        try {
+            //Projects without the war/java plugin won't have this configuration
+            //compileClasspath is not a regular Configuration object so we have to go through without using getAllDependenciesFromConfiguration()
+            project.configurations.getByName('compileClasspath').resolvedConfiguration.resolvedArtifacts.collect { it.moduleVersion.id }.each { ModuleVersionIdentifier id ->
+                logger.debug("Found dependency while processing project: " + id.group.toString() + ":"
+                        + id.name.toString() + ":" + id.version.toString())
+                        
+                dependencies.put(id.group.toString() + ":" + id.name.toString(), id.version.toString())
+            }
+        } catch (UnknownConfigurationException ue) {
+            logger.debug("The compileClasspath configuration was not found.")
+        }
+
+        //Will always have the boostApp configuration since we create it in apply()
+        dependencies.putAll(getAllDependenciesFromConfiguration(project.configurations.boostApp, logger))
+
+        return dependencies
+    }
+
+    public static Map<String, String> getAllDependenciesFromConfiguration(Configuration configuration, BoostLogger logger) {
```

Comment:
I think this method should be private since it is only called from `getAllDependencies`.